workspaces/core/src/banner/private/Multiline.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export interface Props {
+  description: string;
+  title?: string;
+  iconHeader?: React.ReactElement;
+  buttons?: React.ReactElement[];
+}
+
+const Multiline: React.FC<Props> = ({
+  description,
+  title,
+  iconHeader,
+  buttons
+}) => {
+  return (<>
+    {iconHeader && (<div className='w-full bg-goten flex rounded-lg justify-between'>
+      <div className='flex-col p-4'>
+        <div className='flex mb-2 justify-between'>
+          <div className='font-semibold'>{title}</div>
+        </div>
+        <div className='block align-middle items-center text-trunks'>{description}</div>
+      </div>
+      <div className='flex justify-end items-baseline p-4 cursor-pointer'>{iconHeader}</div>
```

Comment:
You put `cursor-pointer` there. Is all this area will have a reaction on click or only click on the close icon? 
![image](https://user-images.githubusercontent.com/59561179/177518129-b8bd2546-6d52-4990-94d0-a7c8f276b409.png)